packages/http/src/mocker/HttpMocker.ts

```diff
@@ -45,8 +45,8 @@ export class HttpMocker
 
     // preparing response body
     let body;
-    if (negotiationResult.example) {
-      body = negotiationResult.example.value;
+    if (negotiationResult.example && (negotiationResult.example as INodeExample).value) {
```

Comment:
Do we allow `value` to be an empty string for instance?

Reply:
Good catch! I changed to check specifically for undefined.